### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RoastEgg**
First version of mtl training and evaluation (can't adapt the interfaces of David).

### CODE REVIEW

File: src/main/java/ml/shifu/shifu/container/obj/ModelTrainConf.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
      * @author Zhang David (pengzhang@paypal.com)
      */
     public static enum ALGORITHM {
-        NN, LR, SVM, DT, RF, GBT, TENSORFLOW, WDL
+        NN, LR, SVM, DT, RF, GBT, TENSORFLOW, WDL, MTL
```
Comment (pylSER) [inline src/main/java/ml/shifu/shifu/container/obj/ModelTrainConf.java]: hailong you are the best!

**Comment by huzza** [comment]
OK... it's a huge PR...

**Comment by huzza** [comment]
For multi target, they use different ColumnConfig.json but same hidden layers. Is that right?

File: src/main/java/ml/shifu/shifu/container/obj/ModelTrainConf.java (hunk 1 above)
Comment (Enaric) [reply src/main/java/ml/shifu/shifu/container/obj/ModelTrainConf.java]: hailong you are the best!

**Comment by RoastEgg** [comment]
> OK... it's a huge PR...

**Pull request closed without merging**